components/studentPortal/layout/Layout.tsx

```diff
@@ -113,7 +108,8 @@ export const Layout: React.FC = ({ children }) => {
       >
         <Sidebar />
       </div>
-      {showOnboardingModal && <FreemiumDialogComponent />}
+      {showOnboardingModal && <FreemiumDialogComponent trigger={false} />}
```

Comment:
that controls whether or not the modal needs you to click on something to trigger its opening or for it to come up automatically.  we need that because it needs to come up automatically on the first load in the layout, but you have to click on the icon in the tooltip for it to show in the tooltip on the sidebar.